source/scss/_pf-component-card.scss

```diff
@@ -17,13 +13,94 @@
   .card-header{
     background: $pf-card-cap-bg;
     border-bottom-color: $pf-card-border-color;
+    padding: $pf-spacer-lg;
+    .card-title {
+      margin: 0;
+    }
+    .dropdown {
+      margin: -$pf-spacer-xxs 0;
+    }
   }
-
   .card-footer{
     background: $pf-card-footer-bg;
-    border-top-color: $pf-card-border-color
+    border-top-color: $pf-card-border-color;
+    padding: $pf-spacer-sm $pf-spacer-lg;
+    p {
+      margin: $pf-spacer-xxxs 0;
+    }
+  }
+  .card-block {
+    padding: $pf-spacer-lg;
   }
+  .card-title {
+    overflow: hidden;
+    text-overflow: ellipsis;
```

Comment:
@andresgalante had made this comment in a previous version of this file:
>I mention this on the CSS, lets add a modifier so users can opt in using it
I'm not sure I understand how this should be implemented. Could we spend some time discussing the next time we chat?